File: Secao08 - Variaveis compostas/Exercicios em Python/Exercicio04.py
# -*- coding: utf-8 -*-
vetor = []
soma = 0

for i in range(1,21):
    num1 = int(input("Informe o {0}º valor: ".format(i)))
    vetor.append(num1)
    soma = soma + num1
print(" ")    
print("A soma de todos os elementos é = {0}".format(soma))

File: Secao08 - Variaveis compostas/Exercicios em Java/Exercicio04.java
package br.com.udemy.secao08;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int[] vetor = new int[20];
		int soma = 0, valor;
		
		for(int i = 0; i < 20; i++) {
			System.out.print("Informe " + (i + 1) + "º valor: ");
			valor = teclado.nextInt();
			vetor[i] = valor;
			
			soma = soma + valor;
		}
		System.out.println(" ");
		System.out.println("Resultado da soma de todos os valores.");
		System.out.println("Soma = " + soma);
		
		teclado.close();
	}

}
